docs/getting-started.md

```diff
@@ -150,6 +164,14 @@ curl --unix-socket /tmp/firecracker.sock -i  \
     }'
 ```
 
+Want to spin up 1000s of microVMs on your bare metal instance?
+
+```
+for ((i=0; i<1000; i++)); do
```

Comment:
This is not posix shell compliant. Better use `for i in $(seq 1000); do`

This also does not start VM. They would still need to be configured and booted

Reply:
Updated

Reply:
@arun-gupta not sure if you didn't push or if github displays an outdated version, but I still see the original version.